src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -162,112 +157,109 @@ Feature: Descriptors are a set of mechanisms to support flexible enumerations or
+                    "type": "urn:ed-fi:api:bad-request:data-validation-failed",
                     "title": "Data Validation Failed",
                     "status": 400,
                     "correlationId": null
                   }
                   """
 
-        # Descriptors are not validating the whitespace yet. DMS-295
-        @ignore
         Scenario: 08 Ensure clients cannot create a descriptor with trailing spaces in required attributes
              When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
                   """
                     {
                         "codeValue": "a                      ",
                         "description": "a                    ",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor   ",
                         "shortDescription": "a                   "
                     }
                   """
              Then it should respond with 400
               And the response body is
-                  """{
+                  """
+                  {
                     "validationErrors": {
                         "$.codeValue": [
-                            "codeValue cannot contain leading or trailing spaces."
+                        "codeValue cannot contain leading or trailing spaces."
                         ],
                         "$.namespace": [
-                            "namespace cannot contain leading or trailing spaces."
-                            ],
+                        "namespace cannot contain leading or trailing spaces."
+                        ],
                         "$.shortDescription": [
-                            "shortDescription cannot contain leading or trailing spaces."
+                        "shortDescription cannot contain leading or trailing spaces."
                         ]
                     },
                     "errors": [],
                     "detail": "Data validation failed. See 'validationErrors' for details.",
-                    "type": "urn:ed-fi:api:bad-request:data",
+                    "type": "urn:ed-fi:api:bad-request:data-validation-failed",
                     "title": "Data Validation Failed",
                     "status": 400,
                     "correlationId": null
                   }
                   """
 
-        @ignore
         Scenario: 09 Post a new descriptor with an extra property (overpost)
             Given a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
                   """
                     {
-                        "codeValue": "Sick Leave",
+                        "codeValue": "Sick Leave2",
                         "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
-                        "shortDescription": "Sick Leave",
+                        "shortDescription": "Sick Leave2",
```

Comment:
Needs to be different than the one we created in the `Background`, so that we're performing an INSERT rather than an UPDATE.